components/CreatedAssets/CreatedAssets.tsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from 'react';
+import { ethers } from 'ethers';
+
+import useEthersProvider from '../../hooks/useEthersProvider';
+import useWeb3Provider from '../../hooks/useWeb3Provider';
+import isLSP7orLSP8 from '../../utils/isLSP7orLSP8';
+import { LSPType } from '../../interfaces/lsps';
+import LSP7Table from '../../components/LSP7Table/LSP7Table';
+import LSP8Table from '../../components/LSP8Table/LSP8Table';
+import fetchIssuedAssets from '../../utils/fetchIssuedAssets';
+
+interface IProps {
```

Comment:
In some files you give a more explicit name such as `LSP7CardProps` or sometimes we only have `Props` and here we have `IProps`. Let's stay consistent. I think using simpply `Props` is the easiest / simplest. For all components.